solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/ModelQuery.java

```diff
@@ -104,10 +107,6 @@ public void setOriginalQuery(Query mainQuery) {
     originalQuery = mainQuery;
   }
 
```

Comment:
`efi` can be marked `final` now then (no need to update this PR for just that though, will include in a future PR)